### PR TITLE
Feature/connection string settings

### PR DESCRIPTION
**Pull request by xtreampb**

### CODE REVIEW

File: source/Calamari.Tests/testSettings.runsettings [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
   <RunConfiguration>
     <EnvironmentVariables>
-      <Azure_OctopusAPITester_SubscriptionId></Azure_OctopusAPITester_SubscriptionId>
-      <Azure_OctopusAPITester_TenantId></Azure_OctopusAPITester_TenantId>
-      <Azure_OctopusAPITester_ClientId></Azure_OctopusAPITester_ClientId>
-      <Azure_OctopusAPITester_Password></Azure_OctopusAPITester_Password>
+      <Azure_OctopusAPITester_SubscriptionId>84d2a09b-50bf-4ad3-98ec-4bc2bec36820</Azure_OctopusAPITester_SubscriptionId>
+      <Azure_OctopusAPITester_TenantId>27312afb-009f-4fed-a8bb-9737425cc42a</Azure_OctopusAPITester_TenantId>
+      <Azure_OctopusAPITester_ClientId>0f91c747-93cf-464e-9a6c-d46c93eef239</Azure_OctopusAPITester_ClientId>
+      <Azure_OctopusAPITester_Password>EU.M~6P3pCHe4K__x3~jif.keOtae5A7Xz</Azure_OctopusAPITester_Password>
```
Comment (MJRichardson) [inline source/Calamari.Tests/testSettings.runsettings:8]: We probably want to remove these!

File: source/Sashimi/Sashimi.csproj.DotSettings [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=actionhandlers/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (MJRichardson) [inline source/Sashimi/Sashimi.csproj.DotSettings:2]: Am I correct in thinking we don't want to commit this file?

File: source/Calamari.Tests/AppServiceSettingsBehaviourFixture.cs [hunk 3]
```diff
@@ -92,7 +92,12 @@ public async Task CleanupCode()
         {
             await resourceGroupClient.StartDeleteAsync(resourceGroupName);
         }
-        
+
+        public async Task TestAppSettingsDeployCommand()
+        {
+
+        }
```
Comment (MJRichardson) [inline source/Calamari.Tests/AppServiceSettingsBehaviourFixture.cs:99]: I'm guessing this is work-in-progress

**Review by MJRichardson — CHANGES_REQUESTED** [review]
I've left a few comments inline, but in general looks good.

File: source/Calamari.Tests/testSettings.runsettings (hunk 1 above)
Comment (xtreampb) [reply source/Calamari.Tests/testSettings.runsettings]: addressed in fb0c1bf

File: source/Sashimi/Sashimi.csproj.DotSettings (hunk 2 above)
Comment (xtreampb) [reply source/Sashimi/Sashimi.csproj.DotSettings:2]: used to mark one of the folders in the project as not a namespace provider

File: source/Calamari.Tests/AppServiceSettingsBehaviourFixture.cs (hunk 3 above)
Comment (xtreampb) [reply source/Calamari.Tests/AppServiceSettingsBehaviourFixture.cs]: updated in a0fdefe

**Review by MJRichardson — APPROVED** [review]

**Pull request merged**